### ISSUE TITLE
Who holds copyright for the tiff and png icons, and under which license?

### ISSUE THREAD

**Issue by sten0** [aquamacs-emacs/aquamacs-emacs#138]
Dear Aquamacs maintainer,

Thank you for maintaining this wonderful port.

Who holds the copyright for the following tiff and png icons, and under which license?
`aquamacs/src/site-lisp/tabbar/*.png`
`aquamacs/src/site-lisp/tabbar/*.tiff`

If I had to guess, I would guess The Aquamacs Project, GPL-3+, but that's not yet clearly documented. 
 Also is there a particular reason the up.png and down.png aren't converted to tiff?  Would you please include all of this information somewhere in the repository?

Kind regards,
Nicholas

**Comment by davidswelt** [comment]
These icons are really quite simple and not very distinctive, but in general, copyright information is here:

aquamacs/Icons/build/COPYING

They are PNG presumably to preserve opacity / alpha channels, although I don’t remember with certainty.  I do think that there are C-level changes to image processing to facilitate Aquamacs’ toolbars and tabbars.  This may get in the way of effectively extracting the tabbar code for use in GNU Emacs, although it is not necessarily a general problem.

David

**Comment by sten0** [comment]
Hi David,

On 26 July 2018 at 16:17, David Reitter <notifications@github.com> wrote:
>
> These icons are really quite simple and not very distinctive, but in
general, copyright information is here:
>
> aquamacs/Icons/build/COPYING

Digging through @dholm's tabbar git history I was able to piece together
```
Files: close-tab.tiff left_disabled.tiff left.tiff right_disabled.tiff
right.tiff
Copyright: 2012 The Aquamacs Project, also © David Reitter
License: GPL-3+

Files: down.png up.png
Copyright: 2008 champo?
License: GPL-2+?  GPL-3+?
```
and after reading aquamacs/Icons/build/COPYING it sounds like the following
is probably correct, assuming that "champo" is Adrian Chromenko:
```
Files: close-tab.tiff left_disabled.tiff left.tiff right_disabled.tiff
right.tiff
Copyright: 2006, 2007, 2008, 2013 David Reitter
          2012, 2015 The Aquamacs Project
License: GPL-3+

Files: down.png up.png
Copyright: 2008 2006, 2007, 2008 by Adrian Chromenko
          2012, The Aquamacs Project
License: GPL-3+
```

>
> They are PNG presumably to preserve opacity / alpha channels, although I
don’t remember with certainty. I do think that there are C-level changes to
image processing to facilitate Aquamacs’ toolbars and tabbars. This may get
in the way of effectively extracting the tabbar code for use in GNU Emacs,
although it is not necessarily a general problem.

Thank you for explaining this :-)  After doing a bit of git history
spelunking, my suspicion that these files had a different copyright was
confirmed.  So aquamacs/Icons/build/COPYING, applies to
`aquamacs/src/site-lisp/tabbar` and not just `aquamacs/Icons/tabbar`?  With
the exception of down.png and up.png, are the icons
in aquamacs/src/site-lisp/tabbar generated from aquamacs/Icons/tabbar?

Once again, thank you very much for helping with this investigation.  The
context is I'm preparing to upload tabbar.el to Debian, and the project is
very strict about copyright.

Cheers,
Nicholas

**Comment by nathaniel** [comment]
Hi Nicholas,
I (Nathaniel Cunningham) am champo (my original sourceforge username). I
created the tabbar icons (not Adrian Chromenko).

-- Nathaniel

**Comment by sten0** [comment]
Hi Nathaniel,

On 26 July 2018 at 18:43, Nathaniel Cunningham <notifications@github.com>
wrote:
>
> Hi Nicholas,
> I (Nathaniel Cunningham) am champo (my original sourceforge username). I
> created the tabbar icons (not Adrian Chromenko).


All the tabbar icons, in both aquamacs/Icons/tabbar and
aquamacs/src/site-lisp/tabbar/?  And you attributed copyright to The
Aquamacs Project, like for aquamacs-tabbar.el?

Cheers,
Nicholas

**Comment by davidswelt** [comment]
Nathaniel, if you like, you can clarify that in the COPYING file along with the other icons.

Is Aquamacs included in the Debian distribution or does this have to do with the Tabbar portion only?

David

**Comment by sten0** [comment]
On 26 July 2018 at 20:08, David Reitter <notifications@github.com> wrote:
>
> Nathaniel, if you like, you can clarify that in the COPYING file along
with the other icons.

That would be very much appreciated :-)

>
> Is Aquamacs included in the Debian distribution or does this have to do
with the Tabbar portion only?

In the emacs-goodies-el package we had included a version of David Ponce's
tabbar.el, with updates from Peter S Galbraith.  I'm not sure if this
Tabbar was forked from Aquamacs, or if it predated Aquamacs.

;; Copyright (C) 2003, 2004, 2005 David Ponce


;; Author: David Ponce <david@dponce.com>

;; Maintainer: David Ponce <david@dponce.com>

;; Created: 25 February 2003

;; Keywords: convenience

;; Revision: $Id: tabbar.el,v 1.2 2007-08-08 22:24:29 psg Exp $

The Debian Emacsen Team recently adopted emacs-goodies-el, and we are
breaking it up into a series of Packages.el-compatible (in Debian speak
"elpafied" via dh-elpa) Debian packages, where each package has a living
upstream.  When searching for a living upstream for tabbar.el, @dholm's
fork of Aquamacs was the defacto choice, given it's popularity on MELPA.

When a team member admonished me for my initial copyright review—I confess
to laziness—I got to work on a full copyright review.  This is a tedious
process that may seem very pedantic to those not involved in Debian...  At
any rate, yes, we're only packaging the Tabbar portion forked by David
Holm, and the FTPMasters (guardians of the archive) require documentation
of copyright, preferably as part of upstream sources; although, a permanent
copies of emails, including headers are also valid, as are emails to the
Debian bug tracker.  This is an interesting case because the chain of
required documentation led to a 2nd degree upstream :-)  So copyright in
Aquamacs should be synced to tabbar, which is then deconstructed in
tabbar-el/debian/copyright.  For free software in Debian, this constitutes
due diligence in providing proof of freedom, :-)  and of course for the
non-free and contrib sections it provides (due diligence in) proof of
non-freedom.  The caveat is that debian/copyright is not legally binding,
because most of us aren't lawyers, but we do the best that we can.

Thank you again for your help on this matter.

Sincerely,
Nicholas
